### ISSUE TITLE
Devices not longer working: "Light does not support that"

### ISSUE THREAD

**Issue by FelixHauser** [Aircoookie/Espalexa#81]
Hi all,

I had a Wemos Mini with 8 devices set up for a long time and working perfectly. All the devices do the same thing send a 433 Mhz signal to turn on/off [radio-enabled sockets](https://www.amazon.de/mumbi-4-Kanal-1100-Funksteckdosen-FS600/dp/B00NQ6FJF6/ref=sr_1_11?crid=FCRGAQ9DRKDB&keywords=funksteckdose&qid=1570817579&sprefix=funk%2Caps%2C174&sr=8-11).

Recently I changed the name of my Wi-Fi at home and since then, some of the devices are not responsive. Usually I only said: "Alexa, light On" and she said "OK", and the light switched on. Now for some of the devices Alexa just tells me "Light does not support that". I hope that's the right translation as my Echo devices are set in German.

I did not change anything other than the WiFi name and Password on my Sketch.

Any help or constructive discussion is welcomed.

**Comment by d-999** [comment]
Hey there, 
are you using the newest version of Espalexa? And do you also have these problems running the example sketch? Also are you using any other libraries in your sketch? And if so which ones?
The OTA library for example doesn't seem to be compatible with Espalexa and leads to similar problems.

**Comment by FelixHauser** [comment]
Yes, I'm using thr latest update. About the example Sketch I don't know because I never tried as my sketch always used to work...
You can have a look at the Sketch I'm using here, as I did a whole template sketch using your library with the only difference that I used libraries to tidy things up and not have everything together on one .ino file

https://github.com/FelixHauser/Easy_Alexa

Something curious occured yesterday in the middle of the night though. I woke up to several Alexa notifications on my phone. Apparently it detected new devices (the same ones) but this time, they work again. Very weird.

I will keep investigating when I get back home in a couple of days.

**Comment by d-999** [comment]
Hey, 
ok, well I guess it's always possible for something to change and your code and/or the library not to work as expected. Amazon seems to have pushed a lot of different updates for Alexa lately, so maybe it has something to do with that...
Anyway, good you got it to work again I guess, though I know it's not as satisfying when you don't fully understand why it works again.

Btw. I'm not the author of Espalexa, in case it seemed that way (credit goes to AirCookie). I just saw the issue you had opened and thought I'd try and help you out...
